tests/mocks/pubclient_stream_trades_function.json

```diff
@@ -1 +1 @@
5-12T15:23:01.53017Z","trade_id":8408612,"price":"453.37000000","size":"0.05070000","side":"sell"},{"time":"2016-05-12T15:23:01.529864Z","trade_id":8408611,"price":"453.37000000","size":"0.03330000","side":"sell"},{"time":"2016-05-12T15:22:34.192663Z","trade_id":8408610,"price":"453.39000000","size":"0.68556000","side":"sell"},{"time":"2016-05-12T15:22:34.19245Z","trade_id":8408609,"price":"453.37000000","size":"0.03610000","side":"sell"},{"time":"2016-05-12T15:22:34.192139Z","trade_id":8408608,"price":"453.37000000","size":"0.04340000","side":"sell"},{"time":"2016-05-12T15:22:03.816277Z","trade_id":8408607,"price":"453.44000000","size":"1.28804000","side":"sell"},{"time":"2016-05-12T15:22:03.816085Z","trade_id":8408606,"price":"453.39000000","size":"0.07500000","side":"sell"},{"time":"2016-05-12T15:22:03.815807Z","trade_id":8408605,"price":"453.38000000","size":"0.10200000","side":"sell"},{"time":"2016-05-12T15:22:02.513243Z","trade_id":8408604,"price":"453.37000000","size":"0.02000000","side":"sell"},{"time":"2016-05-12T15:21:43.977294Z","trade_id":8408603,"price":"453.39000000","size":"0.25623591","side":"sell"},{"time":"2016-05-12T15:21:43.7184Z","trade_id":8408602,"price":"453.39000000","size":"0.25623591","side":"sell"},{"time":"2016-05-12T15:21:43.429748Z","trade_id":8408601,"price":"453.39000000","size":"0.25623591","side":"sell"},{"time":"2016-05-12T15:21:43.173613Z","trade_id":8408600,"price":"453.39000000","size":"0.25623591","side":"sell"},{"time":"2016-05-12T15:21:42.912049Z","trade_id":8408599,"price":"453.39000000","size":"0.25623591","side":"sell"},{"time":"2016-05-12T15:21:35.990748Z","trade_id":8408598,"price":"453.41000000","size":"0.02246000","side":"sell"},{"time":"2016-05-12T15:21:35.990566Z","trade_id":8408597,"price":"453.40000000","size":"0.01000000","side":"sell"},{"time":"2016-05-12T15:21:35.990349Z","trade_id":8408596,"price":"453.38000000","size":"0.03690000","side":"sell"},{"time":"2016-05-12T15:21:35.990059Z","trade_id":8408595,"price":"453.37000000","size":"0.03690000","side":"sell"},{"time":"2016-05-12T15:21:01.234086Z","trade_id":8408594,"price":"453.36000000","size":"0.64393000","side":"buy"},{"time":"2016-05-12T15:21:01.233869Z","trade_id":8408593,"price":"453.36000000","size":"0.01000000","side":"buy"},{"time":"2016-05-12T15:21:01.233699Z","trade_id":8408592,"price":"453.37000000","size":"0.04000000","side":"buy"},{"time":"2016-05-12T15:21:01.233505Z","trade_id":8408591,"price":"453.37000000","size":"0.01000000","side":"buy"},{"time":"2016-05-12T15:21:01.233336Z","trade_id":8408590,"price":"453.38000000","size":"0.04000000","side":"buy"},{"time":"2016-05-12T15:21:01.233146Z","trade_id":8408589,"price":"453.39000000","size":"0.04000000","side":"buy"},{"time":"2016-05-12T15:21:01.23292Z","trade_id":8408588,"price":"453.40000000","size":"0.02400000","side":"buy"},{"time":"2016-05-12T15:21:01.23272Z","trade_id":8408587,"price":"453.40000000","size":"0.01000000","side":"buy"},{"time":"2016-05-12T15:21:01.232543Z","
```

Comment:
Added an return blank endpoint, to simulate a call that returns nothing. The only change is the addition of:

```{"scope":"https://api.gdax.com:443","method":"GET","path":"/products/BTC-USD/trades?before=8409514&after=8409615&limit=100","body":"","status":200,"response":[]}``` 
endpoint which is called by getProductTradeStream() as the next endpoint to call when the first one was consumed.